Sources/ProjectSpec/Scheme.swift

```diff
@@ -150,14 +150,20 @@ public struct Scheme: Equatable {
 
     public struct Archive: BuildAction {
         public var config: String?
+        public var revealArchiveInOrganizer: Bool
+        public var customArchiveName: String?
```

Comment:
The archive name is shown before `revealArchiveInOrganizer` in the dialog. Can you swap these (here and below) to match that?

Reply:
@brentleyjones This is the order of an xcproj's `ArchiveAction` initializer, but I can change it here for sure.